src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,23 @@
-export const UserInfo = () => {};
+interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+}
+
+export type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
+
+  return (
+    <a className="UserInfo" href={`mailto:${email}`}>
```

Comment:
Consider using `rel='noopener noreferrer'` attribute for security reasons when using `target='_blank'` attribute. 🔒